docs-chef-io/content/inspec/resources/docker_image.md

```diff
@@ -11,142 +11,145 @@ platform = "linux"
+This resource is distributed with Chef InSpec.
 
 ### Version
 
-This resource first became available in v1.21.0 of InSpec.
+This resource is available from the InSpec version, 1.21.0.
 
 ## Syntax
 
-A `docker_image` resource block declares the image:
+A `docker_image` resource block declares the image.
 
-    describe docker_image('alpine:latest') do
-      it { should exist }
-      its('id') { should eq 'sha256:4a415e...a526' }
-      its('repo') { should eq 'alpine' }
-      its('tag') { should eq 'latest' }
-    end
+  describe docker_image('ALPINE:LATEST') do
+    it { should exist }
+    its('id') { should eq 'sha256:4a415e...a526' }
+    its('repo') { should eq 'ALPINE' }
+    its('tag') { should eq 'LATEST' }
+  end
 
-## Resource Parameter Examples
+### Resource Parameter Examples
 
-The resource allows you to pass in an image id:
+The resource allows you to pass with an image ID.
 
-    describe docker_image(id: alpine_id) do
-      ...
-    end
+  describe docker_image(id: ID) do
+    ...
+  end
 
-If the tag is missing for an image, `latest` is assumed as default:
+If the tag is missing for an image, `LATEST` is assumed as default.
 
-    describe docker_image('alpine') do
+    describe docker_image('ALPINE') do
       ...
     end
 
-You can also pass in repository and tag as separate values
+You can also pass the repository and tag values as separate values.
 
-    describe docker_image(repo: 'alpine', tag: 'latest') do
-      ...
-    end
+  describe docker_image(repo: 'ALPINE', tag: 'LATEST') do
+    ...
+  end
 
 ## Properties
 
 ### id
 
-The `id` property returns the full image id:
+The `id` property returns the full image ID.
 
-    its('id') { should eq 'sha256:4a415e3663882fbc554ee830889c68a33b3585503892cc718a4698e91ef2a526' }
+  its('id') { should eq 'sha256:4a415e3663882fbc554ee830889c68a33b3585503892cc718a4698e91ef2a526' }
 
 ### image
 
-The `image` property tests the value of the image. It is a combination of `repository/tag`:
+The `image` property tests the value of the image. It is a combination of `repository/tag`.
 
-    its('image') { should eq 'alpine:latest' }
+  its('image') { should eq 'ALPINE:LATEST' }
 
 ### repo
 
-The `repo` property tests the value of the repository name:
+The `repo` property tests the value of the repository name.
 
-    its('repo') { should eq 'alpine' }
+  its('repo') { should eq 'ALPINE' }
 
 ### tag
 
-The `tag` property tests the value of image tag:
+The `tag` property tests the value of the image tag.
 
-    its('tag') { should eq 'latest' }
+  its('tag') { should eq 'LATEST' }
 
-### Low-level information of docker image as `docker_image`'s Property
+### Low-level information of docker image as docker_image's property
 
-#### inspection
-`:inspection` property allows testing the low-level information of docker image returned by `docker inspect [docker_image]`. Use hash format `'key' => 'value` for testing the information.
+#### Inspection
```

Comment:
This should be changed to `inspection` instead of `Inspection`. And line 87 could be removed.